Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Video/GetVideoByIdTest.cs

```diff
@@ -0,0 +1,114 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.MediatR.Media.Video.GetById;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System.Linq.Expressions;
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.Media.Video;
+
+public class GetVideoByIdTest
+{
+    private Mock<IRepositoryWrapper> _mockRepository;
+    private Mock<IMapper> _mockMapper;
+
+    public GetVideoByIdTest()
+    {
+        _mockMapper = new Mock<IMapper>();
+        _mockRepository = new Mock<IRepositoryWrapper>();
+    }
+
+    [Theory]
+    [InlineData(1)]
+    public async Task GetById_ShouldReturnSuccessfullyExistingId(int id)
+    {
+        //Arrange
+        _mockRepository.Setup(x => x.VideoRepository
+            .GetFirstOrDefaultAsync(
+               It.IsAny<Expression<Func<DAL.Entities.Media.Video, bool>>>(),
+                It.IsAny<Func<IQueryable<DAL.Entities.Media.Video>,
+                IIncludableQueryable<DAL.Entities.Media.Video, object>>>()))
+            .ReturnsAsync(new DAL.Entities.Media.Video()
+            {
+                Id = id
+            });
```

Comment:
Move the logic of creation new Entitie outside of return

Reply:
Ok!